_data/tutorials.yaml

```diff
@@ -257,3 +257,14 @@ naming-changelog-repartition-topics:
     ksql: disabled
     kstreams: enabled
     kafka: disabled
+
+console-consumer-produer-basic:
+  title: "Console Consumer and Producer Basics"
+  meta-description: "Produce and consume your first message with Kafka"
+  slug: "/kafka-console-consumer-producer-basics"
+  problem: "you are brand new to Kafka and you want to experiment by producing and consuming a message quickly"
```

Comment:
At some point, there may be another tutorial to do this with Avro (and Protobuf and JSON) with Schema Registry.  Just being cognizant of that, does this naming & problem description take into account those future tutorials?

Reply:
ack